instrumentation/vaadin-14.2/testing/src/main/java/io/opentelemetry/javaagent/instrumentation/vaadin/AbstractVaadin16Test.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.vaadin;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions;
+import io.opentelemetry.sdk.testing.assertj.TracesAssert;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import java.util.List;
+
+public class AbstractVaadin16Test extends AbstractVaadinTest {
+  @Override
+  void assertFirstRequest() {
+    await()
+        .untilAsserted(
+            () -> {
+              List<List<SpanData>> traces = testing.waitForTraces(1);
+              TracesAssert.assertThat(traces)
+                  .satisfies(
+                      trace -> {
+                        assertThat(trace.get(0))
+                            .satisfies(
+                                spans ->
```

Comment:
Wouldn't that be the same as
```suggestion
              List<List<SpanData>> traces = testing.waitForTraces(1);
              assertThat(traces.get(0))
                  .satisfies(
                      spans ->
```
?

Reply:
Thanks, these should be the same.